### ISSUE TITLE
Main world rendered unusable

### ISSUE THREAD

**Issue by fungreenfox** [chaseoes/TF2#23]
What is the changes this plugin does to the main world, making it not dropping items, even if plugin is uninstalled? This has rendered my main world unusable.

For info, i removed everything after have been running TF2. Only main world back, no permissions or any little plugin, any file in plugins folder removed, and STILL can not get any drop from mining anything, a piece of dirt even.
So what is it? TF2 world killer? This will cost me a lot of time rebuilding, or replace with a week old backup.

**Comment by Cysioland** [comment]
Please do following steps:
1. Regenerate world
2. Install TF2 and set it up
3. Check, if problem persists

**Comment by fungreenfox** [comment]
You do not quite understand my problem. My problem is, i stupidly had my main world attached as normal , and installed tf2 to see if i can use it. I expected to find a config allowing me to set which worlds i will use. But there is not such feature. I uninstalled the TF2 plugin and found that nobody can build and get drops anymore in the main world. I made a copy of my server containing 113 plugins and 7 worlds, starting to find an error. I ended up from those 113 plugins down to zero plugins, plugins folder were now empty and the error still persists. I have had to roll the world back 7 days from a backup. Its the dat files in the world folder that are changed somehow.
